demos/checkbox.scss

```diff
@@ -17,4 +17,16 @@
 @import "./common";
 @import "../packages/mdc-checkbox/mdc-checkbox";
 @import "../packages/mdc-form-field/mdc-form-field";
+// TODO(acdvorak): Rename file to `_color-palette`
+@import "../packages/mdc-theme/color_palette";
 @import "../packages/mdc-typography/mdc-typography";
+
+$demo-custom-color: $material-color-red-500;
+
+.demo-checkbox--custom {
+  @include mdc-checkbox-marked-stroke-color($demo-custom-color);
+  @include mdc-checkbox-marked-container-fill-color($demo-custom-color);
```

Comment:
You dont have any "checkbox-unmarked" mixin calls here. We should demonstrate that customizable feature on our demo pages as well.

Reply:
Done